<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gradient</class>
 <widget class="QDialog" name="Gradient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>渐变色填充</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>从模板选择</string>
   </property>
   <widget class="QComboBox" name="comboBoxHis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>填充方向选择</string>
   </property>
   <widget class="QPushButton" name="pushButtonLeftToRight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTopToBottom">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLeftBottomToRightTop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLeftTopToRightBottom">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRadient">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>预览</string>
   </property>
   <widget class="QLabel" name="LabelPeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>渐变调整</string>
   </property>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>左键单击增加/右键单击三角形删除</string>
    </property>
    <widget class="QPushButton" name="pushButtonHeadToTail">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelMultiColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>颜色选择</string>
    </property>
    <widget class="QPushButton" name="pushButtonColorPick">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>透明度</string>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>367</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>367</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonColorPick</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>pushButtonHeadToTail</tabstop>
  <tabstop>pushButtonRadient</tabstop>
  <tabstop>pushButtonLeftTopToRightBottom</tabstop>
  <tabstop>pushButtonLeftBottomToRightTop</tabstop>
  <tabstop>pushButtonTopToBottom</tabstop>
  <tabstop>pushButtonLeftToRight</tabstop>
  <tabstop>comboBoxHis</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
